tests/backup_restore_test.go

```diff
@@ -81,6 +81,10 @@ func TestServer_BackupAndRestore(t *testing.T) {
 			t.Fatalf("query results wrong:\n\texp: %s\n\tgot: %s", expected, res)
 		}
 
+		// it takes some time for _internal to be created, we want to make sure it's treated
+		// correctly.
+		time.Sleep(time.Second * 10)
```

Comment:
You might want to bump this a little to be on the safe side. What with goroutine scheduling and all, it might not be exactly 10 seconds.